lib/child_process.js

```diff
@@ -781,7 +782,7 @@ function spawn(file, args, options) {
     if (signal.aborted) {
       process.nextTick(onAbortListener);
     } else {
-      signal.addEventListener('abort', onAbortListener, { once: true });
+      signal.addEventListener('abort', onAbortListener, { once: true, [kForcedHandler]: true });
```

Comment:
```suggestion
      signal.addEventListener('abort', onAbortListener, { __proto__: null, once: true, [kForcedHandler]: true });
```

Reply:
Reverted this for now due to @benjamingr's comment 🙂